src/components/StatusBar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+interface StatusBarProps {
+    title: string;
+    progressMax: number;
+    progressNow: number;
+}
+
+export function StatusBar(props: StatusBarProps) {
+    return (
+        <div className="flex flex-col items-center space-y-4">
+            <h1 className="text-xl font-bold text-white">{props.title}</h1>
+            <div className="flex items-center">
+                {Array.from({ length: props.progressMax }).map((_, index) => (
+                    <React.Fragment key={index}>
```

Comment:
<></> short hand for fragments, 
But doesn't this work without out it, just one div?

Reply:
yeah, actually does, overcomplicated & hence simplified